t/tie.t

```diff
@@ -0,0 +1,29 @@
+use strict;
+use warnings;
+
+use lib 't/lib';
+
+use TestHelper qw( doc );
+use Test::More import => [qw( done_testing is $TODO )];
+
+my $test_data_filename = 'test-data/tie.pl';
+my ($doc) = doc(
+    filename        => $test_data_filename,
+    preserve_unused => 0,
+);
+
+open my $test_data, '<', $test_data_filename
+    or die "Could not open $test_data_filename: $!";
+my $expected = do {
+    local $/;
+    <$test_data>;
+};
+close $test_data
+    or die "Error on closing $test_data_filename: $!";
+
+TODO: {
+    local $TODO = 'Cannot yet see packages used by tie';
+    is( $doc->tidied_document, $expected, 'tie observed' );
+}
```

Comment:
I think for this we could follow the pattern in `t/via-sub-exporter-exception.pl` using `Path::Tiny`.

Reply:
Thanks.  I thought there'd probably be some existing pattern like that, but hadn't been able to find it.  Guess I was grepping for "File" rather than "Path".